lib/amqp.js

```diff
@@ -145,16 +144,34 @@ AMQPConnection.prototype.sendToExchange = function sendToExchange(exchangeName,
     }
 };
 
+AMQPConnection.prototype.prepareMessageAndSendToExchange
```

Comment:
Changing vars to const during implementation is one thing, but rewriting a working code just because you don't like the style is another thing. I don't see any business value in that and see it as waste of time. As I said, it can be done later by somebody else who is more religious about code style that me.